tests/ci/run_benchmark_build_tests.sh

```diff
@@ -13,47 +13,54 @@ openssl_1_0_branch='OpenSSL_1_0_2-stable'
 # failing our CI.
 openssl_3_0_branch='openssl-3.0.5'
 
+function build_aws_lc_fips {
+  echo "building aws-lc in FIPS mode"
+  run_build -DCMAKE_INSTALL_PREFIX="${install_dir}/aws-lc-fips" -DFIPS=1
+  pushd "$BUILD_ROOT"
+  ninja install
+  popd
+}
+
 function build_openssl_1_0 {
     echo "building OpenSSL 1.0"
-    git clone --branch "${openssl_1_0_branch}" "${openssl_url}" "../openssl-1.0"
+    git clone --depth 1 --branch "${openssl_1_0_branch}" "${openssl_url}" "../openssl-1.0"
     pushd "../openssl-1.0"
     mkdir -p "${install_dir}/openssl-1.0"
     ./config --prefix="${install_dir}/openssl-1.0" --openssldir="${install_dir}/openssl-1.0"
-    make "-j${NUM_CPU_THREADS}"
-    make install
+    make "-j${NUM_CPU_THREADS}" > /dev/null
+    make install_sw
     popd
     rm -rf "../openssl-1.0"
 }
 
 function build_openssl_1_1 {
     echo "building OpenSSL 1.1"
-    git clone --branch "${openssl_1_1_branch}" "${openssl_url}" "../openssl-1.1"
+    git clone --depth 1 --branch "${openssl_1_1_branch}" "${openssl_url}" "../openssl-1.1"
     pushd "../openssl-1.1"
     mkdir -p "${install_dir}/openssl-1.1"
     ./config --prefix="${install_dir}/openssl-1.1" --openssldir="${install_dir}/openssl-1.1"
-    make "-j${NUM_CPU_THREADS}"
-    make install
+    make "-j${NUM_CPU_THREADS}" > /dev/null
+    make install_sw
     popd
     rm -rf "../openssl-1.1"
 }
 
 function build_openssl_3_0 {
     echo "building OpenSSL 3.0"
-    git clone --branch "${openssl_3_0_branch}" "${openssl_url}" "../openssl-3.0"
+    git clone --depth 1 --branch "${openssl_3_0_branch}" "${openssl_url}" "../openssl-3.0"
     pushd "../openssl-3.0"
     mkdir -p "${install_dir}/openssl-3.0"
     ./Configure --prefix="${install_dir}/openssl-3.0" --openssldir="${install_dir}/openssl-3.0"
-    make "-j${NUM_CPU_THREADS}"
-    make install
+    make "-j${NUM_CPU_THREADS}" > /dev/null
+    make install_sw
     popd
     rm -rf "../openssl-3.0"
 }
 
 # We build each tool individually so we can have more insight into what is failing
-mkdir -p "${install_dir}"
-echo "Testing awslc_bm"
-mkdir -p "${install_dir}/aws_lc"
-run_build -DAWSLC_INSTALL_DIR="${install_dir}/aws_lc"
+build_aws_lc_fips
+echo "Testing awslc_bm with AWS-LC FIPS"
+run_build -DAWSLC_INSTALL_DIR="${install_dir}/aws-lc-fips"
```

Comment:
Why is this needed when we just did `build_aws_lc_fips` 2 lines above?

Reply:
`build_aws_lc_fips` builds aws-lc and installs the public header files and libcrypto artifact into `${install_dir}/aws-lc-fips`, this line then builds aws-lc again and uses those public artifacts to make the `awslc_bm` executable. This is attempting to replicate building the fips branch with the latest speed.cc with any changes/additions to speed.cc.

Reply:
Please add a comment on what is going on here :)